demos/README.txt

```diff
@@ -18,6 +18,9 @@ EVP_MD_demo.c          Compute a digest from multiple buffers
 EVP_MD_stdin.c         Compute a digest with data read from stdin
 EVP_f_md.c             Compute a digest using BIO and EVP_f_md
 
+encrypt:
+rsa_encrypt.c          Encrypt and Decrypt data using an RSA keypair.
```

Comment:
Lower case "Decrypt" since it is mid sentence?

Reply:
I also noticed this. But I think Shane might want to highlight the functionality of this demo ;-)